src/main/scala/com/github/catalystcode/fortis/spark/streaming/facebook/FacebookUtils.scala

```diff
@@ -10,23 +10,35 @@ import org.apache.spark.streaming.StreamingContext
 import org.apache.spark.streaming.dstream.ReceiverInputDStream
 
 object FacebookUtils {
-  def createPageStream(
+  def createPageStreams(
     ssc: StreamingContext,
     auth: FacebookAuth,
-    pageId: String,
+    pageIds: Set[String],
     fields: Set[String] = Set("message", "place", "caption", "from", "name", "comments"),
```

Comment:
let's remove comments as this should be populated from `createPageCommentsStream`